diff --git a/dist/index.js b/dist/index.js
index 1781dd86094f2936b1f15c84b1c827cdc1875fd6..ea26d4c3b5d996e5fb393f59b59729edaadf195a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -141,8 +141,9 @@ export const HEAD = handle(app);
       join(outputDirectory, this.outputDir, "vercel.json"),
       JSON.stringify(
         {
-          version: 2,
-          installCommand: "npm install --omit=dev",
+          "$schema": "https://openapi.vercel.sh/vercel.json",
+          "version": 2,
+          "installCommand": "npm install --omit=dev",
           builds: [
             {
               src: "index.mjs",
@@ -152,8 +153,8 @@ export const HEAD = handle(app);
           ],
           routes: [
             {
-              src: "/(.*)",
-              dest: "index.mjs"
+              src: "/.*",
+              dest: "/index.mjs"
             }
           ]
         },
