diff --git a/dist/index.js b/dist/index.js
index 1781dd86094f2936b1f15c84b1c827cdc1875fd6..cb2d84d43ae6a4affe9674f88e4f867d2969655f 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -141,8 +141,9 @@ export const HEAD = handle(app);
       join(outputDirectory, this.outputDir, "vercel.json"),
       JSON.stringify(
         {
-          version: 2,
-          installCommand: "npm install --omit=dev",
+          "$schema": "https://openapi.vercel.sh/vercel.json",
+          "version": 2,
+          "installCommand": "npm install --omit=dev",
           builds: [
             {
               src: "index.mjs",
@@ -150,10 +151,10 @@ export const HEAD = handle(app);
               config: { includeFiles: files }
             }
           ],
-          routes: [
+          rewrites: [
             {
-              src: "/(.*)",
-              dest: "index.mjs"
+              src: "/.*",
+              dest: "/index.mjs"
             }
           ]
         },
